### DISCUSSION TITLE
What's the meaning of zm "Align view middle"

### DISCUSSION THREAD

**Discussion by dlintw** [helix-editor/helix#9032] [Q&A] [answered]
I know the meaning of `zc` center.  Why there is another middle key and it seems not work

**Comment by gabydd** [comment, marked as answer]
If you don't have softwrap on and there is enough text on the left it will align your cursor to the middle horizontally

**Reply by dlintw** [reply]
So, maybe the description is better say horizontally.  Thanks.
